### DISCUSSION TITLE
Randomising App List

### DISCUSSION THREAD

**Discussion by thenewlegend** [tanujnotes/Olauncher#264] [Ideas]
# Make Life harder to  Reduce Distraction

I have disabled auto-keyboard pop up. But the app list is in alphabetic order and I have grown familiar with  position of the distracting apps. One scroll to reach insta , Two scroll for Fb ec.

![mk-wiles-mary-kate-wiles](https://user-images.githubusercontent.com/63841336/205831440-8215c7cc-cfec-48d2-b3ca-e6eb1383b393.gif)

### How about adding an option to **randomize the app list each time screen is locked**  ?

This way we have to consciously search for apps we actually want to use and not just use it just because we can. 

![cat-cute](https://user-images.githubusercontent.com/63841336/205832754-936a391c-0d2e-4f69-8352-11c7d1fa1b35.gif)

**Comment by tanujnotes** [comment]
Not everyone will be onboard with this.

You can try hiding your social apps in Olauncher. Finding hidden apps is intentionally made inconvenient for this exact purpose.

Long press on the app in the list for HIDE option.

**Reply by thenewlegend** [reply]
I agree with that but if it is not too complex to pull off maybe include as an option which can be toggled on/off from the main menu.

**Reply by tanujnotes** [reply]
Sorry but that's not how it works. I can't add a feature every time someone asks for it. It stays the way it is.

But since it is open source, you can fork and make the changes yourself or pay someone else to do.

**Comment by thenewlegend** [comment]
Yeah, that's the power of open sourced projects. Doesn't really matter who started it as everyone can  make it their own.

Thanks for the reply though.
